ros/src/computing/perception/localization/packages/twist_generator/src/vehicle_status_converter.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ros/ros.h>
+
+#include <std_msgs/Float32MultiArray.h>
+#include <geometry_msgs/TwistStamped.h>
+#include <autoware_msgs/VehicleStatus.h>
+#include <nav_msgs/Odometry.h>
+
+static const double KMPH2MPS = 1000.0 / 3600.0;
+static const double DEG2RAD = 3.141592 / 180.0;
+
+class VehicleStatusConverter
```

Comment:
why a single file implementation?

Reply:
split this file into .hpp and .cpp. pls check it.